### DISCUSSION TITLE
A thank you to the people who created Spiral Linux

### DISCUSSION THREAD

**Discussion by jshellshear** [SpiralLinux/SpiralLinux-project#89] [General]
Good afternoon. 

I am a new user to Spiral Linux and I wanted to express my gratitude at those who created and tested this distribution including those who made it available to end users such as myself.  

I have tried many differing linux distributions over time and have always found a distinct liking for Ubuntu, Fedora, and a well known variant Linux Mint. Despite liking these distributions I have encountered issues with all of them over time that have left me dissatisfied and on the continual lookout for something that would encapsulate all the things I wanted to find in a linux OS that would stand the test of time. 

Since trying Spiral Linux (Budgie desktop) I have been pleasantly surprised by it's thoughtful design and speed when used on older hardware, especially using the budgie desktop as this is not normally a light desktop to use. 

I am a user that likes to use older hardware for many personal reasons always like to see how that hardware will fare with changing and evolving operating systems. So I have tried spiral linux on 5 differing hardware types, 4 laptops and one desktop. 

Dell M90 laptop
HP 8710p laptop
Dell E6410 laptop 
Gateway NV59C
Dell Precision Xeon desktop

All devices run minimum of 4mb Ram and use hard drives ranging from 5400rpm to 7200rpm.

Spiral linux works well on all devices save the HP 8710p laptop as this has an NVidia video card and requires specific drivers to function well, as without these there are constant differing issues occurring. (Until I resolve this to my satisfaction, I have shelved this unit for the moment)

One thing I have noticed on all devices using Spiral is the speed difference in regular usage compared to Ubuntu, Fedora or even their variants (Linux Mint and Ultramarine Linux) the difference is very noticeable when I run them side by side. Spiral linux literally outpaces them and is much more responsive even under heavy application/s usage.  

So far Spiral linux has been a pleasure to use and any issues I have experienced are not severe or worth complaining about. 

The point of this post is simply to express my sincere gratitude for creating a distribution that is stable and runs well on older hardware. Thank you all for your time and efforts. 

Best regards.

**Comment by geckolinux** [comment]
Hi there @jshellshear , thanks so much for taking the time to write and share your experience, I'm thrilled to hear that it's working well for your needs! Best regards.

**Reply by LordJester** [reply]
I just want to thank YOU for creating this project!
I'm a linux enthusiast for a several years now. I'm buying old laptops and running and testing various distros for the future serious desktop use (haven't make the courage to make that step yet). I'm seeing this project like a neat Debian installer with some nice features besides that. I have installed it ten days ago and I'm very very pleased with it. Just wanted you to know that you make me very happy with this project and I'm very grateful for your time and effort. 
Kruno

**Reply by geckolinux** [reply]
> I'm seeing this project like a neat Debian installer with some nice features besides that.

That's _exactly_ the idea! Thanks a lot, so glad to hear that you like it.

**Comment by gcvl** [comment]
I second that!
I tried and used any Debian derivative and spin and flavour, from Linux Mint DE, MX Linux, Debian itself, antiX, Titan Linux, SparkyLinux, Devuan and now SpiralLinux XFCE.
I really like how you've been able to put together such a pretty, usable and sensible set-up; aside of a couple of custom themes, some packages and a handful of very minor tweaks (mainly systemd related damage control), I was kind of ready to go just out of the box.
Thank you for this well crafted spin and keep it beating!
Cheers
